src/backend/utils/misc/guc.c

```diff
@@ -2865,7 +2866,7 @@ static struct config_string ConfigureNamesString[] =
 
 	{
 		{"temp_tablespaces", PGC_USERSET, CLIENT_CONN_STATEMENT,
-			gettext_noop("Sets the tablespace(s) to use for temporary tables and sort files."),
+			gettext_noop("Sets the tablespace(s) to use for temporary tables."),
```

Comment:
why did you remove last part?

Reply:
Brought it back, and added a description to `temp_file_tablespaces`